src/main/java/com/hackathon03/codeguru/CodeguruApplication.java

```diff
@@ -0,0 +1,18 @@
+package com.hackathon03.codeguru;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class CodeguruApplication {
+
+	public static void main(String[] args) {
+		//S1
+		if("1" == "2"){
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**: An equality operator (== or !=) is used to compare strings, which matches strings based on address.

**Fix**: Do content comparison using the equals() method to compare the values of the strings.